### PR TITLE
ci: implement pipeline to build and publish FE image

### PR DESCRIPTION
**Pull request by anversa-pro**
## Issue ticket number and link
https://trello.com/c/gH2Le7Nm

## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [x] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

-

## What is the new behavior?

-  Implement a new GitHub Actions workflow dedicated to building and pushing frontend Docker images to the GitHub Container Registry upon merges to the master branch.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .github/workflows/build_and_publish.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+       - master
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
```
Comment (jose-sal-y-rosas) [inline .github/workflows/build_and_publish.yml:17]: Please could you use a specific version: 22.04 o 20.04?

**Review by jlora-mck — APPROVED** [review]

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: .github/workflows/build_and_publish.yml (hunk 1 above)
Comment (anversa-pro) [reply .github/workflows/build_and_publish.yml]: Modified as specified

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**